### PR TITLE
Bump kubemacpool deps

### PR DESCRIPTION
**Pull request by rhrazdil**
Bumping versions of used dependencies.

**What this PR does / why we need it**:

**Special notes for your reviewer**:

**Release note**:

```release-note
Bump kubernetes to v0.19.1
Bump github.com/go-logr/logr to v0.2.1
Bump goveralls to v0.0.7
Bump gingko/gomega to v1.14.1/v1.10.2
Bump kubevirt and kubevirt/client-go to v0.32.0
Bump sigs.k8s.io/controller-runtime to v0.6.2
Bump sigs.k8s.io/controller-tools to v0.4.0
Bump kustomize to v0.8.2
Bump CNAO to apiversion v1, release v0.42.1
Bump kubevirtci to k8s-1.19
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -17,8 +17,8 @@ require (
 	k8s.io/apimachinery v0.19.1
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/code-generator v0.19.1
-	kubevirt.io/client-go v0.29.0
-	kubevirt.io/kubevirt v0.29.0
+	kubevirt.io/client-go v0.32.0
+	kubevirt.io/kubevirt v0.32.0
```
Comment (RamLavi) [inline go.mod:21]: need to also update kubevirt in [cluster-up script](https://github.com/k8snetworkplumbingwg/kubemacpool/blob/7600ea1e8a945a5665071bdc7e6d8619adb65195/cluster/up.sh#L28)

**Review by RamLavi — CHANGES_REQUESTED** [review]
we also need to :
- bump kubeivrt in [cluster-up script](https://github.com/k8snetworkplumbingwg/kubemacpool/blob/7600ea1e8a945a5665071bdc7e6d8619adb65195/cluster/up.sh#L28)
- bump CNAO to v1 in [cna-cr.yaml](https://github.com/k8snetworkplumbingwg/kubemacpool/blob/master/hack/cna/cna-cr.yaml) (in another commit)

**Comment by rhrazdil** [comment]
depends on https://github.com/kubevirt/cluster-network-addons-operator/pull/553 to be merged

**Comment by RamLavi** [comment]
> depends on [kubevirt/cluster-network-addons-operator#553](https://github.com/kubevirt/cluster-network-addons-operator/pull/553) to pass the CI

do you mean that this PR need to be merged in order for [kubevirt/cluster-network-addons-operator#553](https://github.com/kubevirt/cluster-network-addons-operator/pull/553) to pass **its** CI?

**Comment by RamLavi** [comment]
@qinqon do you think there is another dependency worth bumping?

**Comment by rhrazdil** [comment]
@RamLavi  The other way around. kubevirt/cluster-network-addons-operator#553 needs to get in in order for this to PR to pass CI

File: go.mod [hunk 2]
```diff
@@ -3,48 +3,49 @@ module github.com/k8snetworkplumbingwg/kubemacpool
...
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.18.2
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.18.2
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.18.2
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.18.2
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.18.2
-	k8s.io/kubelet => k8s.io/kubelet v0.18.2
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.18.2
-	k8s.io/metrics => k8s.io/metrics v0.18.2
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.18.2
+	// Pinned to kubernetes-0.19.1
```
Comment (qinqon) [inline go.mod:29]: 1.19.1

Comment (rhrazdil) [reply go.mod]: Fixed

**Comment by phoracek** [comment]
We need to get https://github.com/kubevirt/project-infra/pull/635 in first

**Comment by phoracek** [comment]
@rhrazdil since the kubevirtci issue will take some time to resolve, could you drop it from this PR? Sorry for the inconvenience.

**Comment by qinqon** [comment]
@phoracek fix has just merged we can update kubevirtci again and it should work

**Comment by phoracek** [comment]
Excellent! Thanks for the update.

**Pull request merged**